src/frappe.json

```
@@ -1,40 +1,88 @@
 {
     "manifest": {
-        "author": "Isabelinc#9113",
-        "name": "Catppuccin Frappe",
-        "version": "1.3.0",
+        "author": "nyraki",
+        "name": "catppuccin frappé",
```

Feels like we should keep the name the same as the existing themes
```suggestion
        "name": "Catppuccin Frappe",
```